lib/_stream_writable.js

```diff
@@ -687,7 +687,6 @@ function endWritable(stream, state, cb) {
       onFinished(stream, state, cb);
   }
   state.ended = true;
-  stream.writable = false;
```

Comment:
@mcollina: The point is to remove this. It’s unnecessary (it should already be false through the computed getter) and causes the ‘writable’ property to be unnecessarily added on the state object.